auth/utils.go

```diff
@@ -0,0 +1,23 @@
+package auth
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+	"log"
+)
+
+// randBytes is the number of bytes of entropy for SA password
+const randBytes = 16
+
+// NewPassword generates cryptographically secure random bytes, base64
+// encodes it, and returns it.
+func NewPassword() string {
+	c := randBytes
+	b := make([]byte, c)
```

Comment:
Why use c at all? why not just randBytes

Reply:
no raisin. It was some copypasta.